addons/account_edi_ubl_cii/data/ubl_20_templates.xml

```diff
@@ -0,0 +1,412 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="ubl_20_AddressType">
+        <t xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
+           xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+           xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+            <cbc:StreetName t-out="vals.get('street_name')"/>
+            <cbc:AdditionalStreetName t-out="vals.get('additional_street_name')"/>
+            <cbc:CityName t-out="vals.get('city_name')"/>
+            <cbc:PostalZone t-out="vals.get('postal_zone')"/>
+            <cbc:CountrySubentity t-out="vals.get('country_subentity')"/>
+            <cbc:CountrySubentityCode t-out="vals.get('country_subentity_code')"/>
+            <cac:Country>
+                <t t-set="country_vals" t-value="vals.get('Country_vals', {})"/>
```

Comment:
the variables with caps are dictionaries (because they refer to a 'type' in datypic) while the lower case variables aren't (they are string, float...); they are "terminal" values in the xml nodes. Should I put every variables lower case with underscores (it's the convention but we lose a bit of info) ?

Reply:
Well, I'll follow the convention